src/websocket_broker/ws_handle.go

```diff
@@ -147,14 +154,20 @@ func (broker *WebSocketBroker) RemoveHandler(topic string, handlerName string) {
 }
 
 func (broker *WebSocketBroker) closeClient(id string) error {
-	broker.clientsMx.Lock()
```

Comment:
This is error prone, if we forget to have a lock of the mutex before calling closeClient